### PR TITLE
Apply v0.2 styling globally

### LINKED ISSUE #94: Fix frontend -> trade-executor dependency management
`frontend` depends on `trade-executor-frontend`; the current dependency management approach is unpredictable and fragile.

## Current Approach

The current approach involves installing `trade-executor-frontend` as direct sibling of `frontend`. Paths defined in `frontend` assume that `trade-executor-frontend` exists in sibling directory.

## Issues

This approach does not preserve any commit-specific references. Any breaking changes to `trade-executor-frontend` make it difficult to install a working version of `frontend`. This will eventually bite us in production – someone will push an upgrade to `frontend` that pulls in the latest (and incompatible) version of `trade-executor-frontend`.

In addition, this approach makes [Containerize frontend](https://github.com/tradingstrategy-ai/scratchpad/issues/6) overly complex to implement, and will result in a fragile container build process.

## Proposed Solution

* Use SvelteKit's built-in `npm package` feature to package up releases of `trade-executor-frontend`
* Build and publish releases to github's npm registry on:tag (in GH action)
* within `frontend`, manage `trade-executor-frontend` like any other npm dependency (`npm install` the correct version)

If this approach turns out to be difficult, we can consider using a `git submodule`.

### PR DESCRIPTION
**Pull request by allozaur**
Close tradingstrategy-ai/design-system#94, tradingstrategy-ai/design-system#95, tradingstrategy-ai/design-system#103

### CODE REVIEW

File: src/lib/components/Timestamp.svelte [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
```
Comment (allozaur) [inline src/lib/components/Timestamp.svelte:1]: Add component docs

File: src/lib/components/css/table.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
+table:not(.table) {
```
Comment (allozaur) [inline src/lib/components/css/table.css:1]: These `:not(.table)` modifiers are there temporarily until we fully migrate from legacy theme

Comment (allozaur) [inline src/lib/components/css/table.css:1]: Missing docs comment

File: src/lib/components/css/truncate.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.truncate {
```
Comment (allozaur) [inline src/lib/components/css/truncate.css:1]: Missing docs comment

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -3,15 +3,16 @@
...
+	import HomeHeroBanner from './home/HeroBanner.svelte';
+	import { BlogRoll, ContentTile, Button, Section, SummaryBox, TopTradesTable } from '$lib/components';
 	import { toggleSubscribeDialog } from '$lib/newsletter/controller';
 	import { sitelinksSearchBox } from '$lib/helpers/googleMeta';
 
 	export let data: PageData;
 
 	const { topMomentum, impressiveNumbers, posts } = data;
+
+	$: console.log('topMomentum', topMomentum);
```
Comment (allozaur) [inline src/routes/+page.svelte:15]: Unneeded code.

```suggestion
```

File: src/routes/home/HeroBanner.svelte [hunk 5]
```diff
@@ -0,0 +1,110 @@
+<!--
+@component
+Home page hero banner.
+
+#### Usage:
+```tsx
+<HeroBanner {impressiveNumbers} />
+```
+-->
```
Comment (allozaur) [inline src/routes/home/HeroBanner.svelte:9]: Update component docs

File: src/routes/qr/+page.svelte [hunk 6]
```diff
@@ -19,29 +19,29 @@ on the conference rolls.
...
+		<ContentTileOld title="Newsletter" icon="newspaper" on:click={toggleSubscribeDialog}>
 			Subscribe to our newsletter and never miss protocol updates, trading tips, news and insights.
-		</ContentTile>
+		</ContentTileOld>
 
-		<ContentTile title="YouTube" icon="youtube" targetUrl="https://www.youtube.com/channel/UCXBQRclPxMY40n52-k3VhYQ">
+		<ContentTileOld title="YouTube" icon="youtube" targetUrl="https://www.youtube.com/channel/UCXBQRclPxMY40n52-k3VhYQ">
 			Subscribe to our Youtube channel and watch the latest videos on DeFi and algorithmic trading.
-		</ContentTile>
+		</ContentTileOld>
```
Comment (allozaur) [inline src/routes/qr/+page.svelte:44]: Refactor this to use `ContentCard` component.

File: src/routes/strategies/[strategy]/(nav)/[status=positionStatus]-positions/+page.svelte [hunk 7]
```diff
@@ -48,6 +48,6 @@
 		display: grid;
 		gap: 1.5rem;
 		align-content: start;
-		overflow: auto;
+		/* overflow: auto; */
```
Comment (allozaur) [inline src/routes/strategies/[strategy]/(nav)/[status=positionStatus]-positions/+page.svelte:51]:
```suggestion
```

**Review by allozaur — COMMENTED** [review]
Alright, @kenkunz! I think this PR is ready for your review :) there still are some things that need to be handled (anything i've managed to find is listed in the threads above). 

Besides these thigs also some tests are failing (probably due to some naming changes) so this also needs to be addressed before merging this PR.

**Comment by allozaur** [comment]
@kenkunz this is ready for your review, all things are looking good from UI standpoint ;)

**Review by kenkunz — APPROVED** [review]
I think this is finally good-to-go 😄

**Comment by miohtama** [comment]
Please ship it 👍

**Pull request merged**